A32NX/html_ui/Pages/A32NX_Core/A32NX_ATSU.js

```diff
@@ -134,12 +134,12 @@ const lbsToKg = (value) => {
  * @param {() => void} updateView
  */
 const getSimBriefOfp = (mcdu, updateView) => {
-    const simBriefUsername = NXDataStore.get("CONFIG_SIMBRIEF_USERNAME", "");
+    const simBriefUsername = NXDataStore.get("CONFIG_SIMBRIEF_USERNAME", ""); // TODO legacy, remove once most users have transitioned away
```

Comment:
why even keep it in the first place?

Reply:
So that everybody who started using this feature when it was first implemented (username only at that point) doesn't need to re-enter their details.